.gitignore

```diff
@@ -365,3 +365,6 @@ gradle-app.setting
 **/build/
 
 # End of https://www.toptal.com/developers/gitignore/api/node,java,yarn,macos,linux,windows,intellij,java-web,visualstudiocode,gradle
+
+.idea/dataSources.xml
+backend/src/main/resources/application-dev.properties
```

Comment:
All of the other custom rules are added to the top of the file and the second one is a duplicate.

Reply:
Right, that slipped in because I didn't see the rebase that brought the duplicate in - fixed